docs/src/tut/community-insurance/index.md

```diff
@@ -0,0 +1,213 @@
+# {#tut} Community insurance
+
+This tutorial walks through the creation of a Reach decentralized application with React (v16.8 - Hooks) as the frontend.
+It demonstrates how you can use reach together with react to build a real world community insurance application.
+To follow along, you are expected to have already gone through [the most basic tutorial](##tut) and you have already
+learnt how to develop and test with a command-line frontend. 
+We highly encourage you to read about [the reach architecture](https://docs.reach.sh/rsh/#ref-programs) as well, 
+as it's an excellent resource to help you understand the different modes/states of any Reach program.
+
+## Assumptions
+This tutorial assumes that you already
+ * have [Docker](https://www.docker.com/get-started), and [Docker Compose](https://docs.docker.com/compose/install/) installed.
+ * have node & npm installed on your machine. If not, first head over to the [official website](https://nodejs.org/en/download/) 
+    and follow the installation steps.
+ * have working knowledge of react hooks. You can checkout [the basics](https://reactjs.org/docs/hooks-overview.html#:~:text=Hooks%20are%20functions%20that%20let,if%20you'd%20like.) in case this is new to you.
+ * installed Reach successfully by following the [installation process](##tut) from the basic tutorial. 
+
+Since this is a real world application, we want to style it professionally using [tailwind css](https://tailwindcss.com/). You don't have to be familiar with this framework to follow along though. 
```

Comment:
```suggestion
Since this is a real-world application, we want to style it professionally using [Tailwind CSS](https://tailwindcss.com/). You don't have to be familiar with this framework to follow along, though. 
```